### PR TITLE
build: update to lnd v0.16.2

### PR DESCRIPTION
**Pull request by Roasbeef**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ The following table shows the supported combinations:
 
 | LiT              | LND          | Loop         | Faraday       | Pool         |
 |------------------|--------------|--------------|---------------|--------------|
+| **v0.9.2-alpha** | v0.16.2-beta | v0.23.0-beta | v0.2.11-alpha | v0.6.2-beta  |
```
Comment (guggero) [inline README.md:137]: nit: needs an entry in the above compatibility table as well.

**Review by guggero — APPROVED** [review]
LGTM :tada:

File: README.md (hunk 1 above)
Comment (ellemouton) [reply README.md:138]: added 👍

**Review by ellemouton — APPROVED** [review]
LGTM 🏃

**Comment by ellemouton** [comment]
cool - gonna merge since it is only an LND bump. Did a quick test run to check that calls to lit, loop, pool, faraday, LND and autopilot calls work

**Pull request merged**